package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Is package-lock.json necessary?

Reply:
No, `package-lock.json` isn't necessary at any level since we are using pnpm. The idea behind pacakge.json is to extract commonly used dev dependencies into a single place